### DISCUSSION TITLE
Is there any kind person who can explain termux working? i plan to bring it to flutter however

### DISCUSSION THREAD

**Discussion by ClarissaGazalaEvanthe** [termux/termux-app#3553] [Q&A]
I'm an amateur developer trying to build termux with flutter, but when I access the shell I get denied even though I run the exe in my folder, and one more thing how does chroot work?

please help me i need a little enlightenment

**Comment by agnostic-apollo** [comment]
If your `targetSdkVersion` is `> 28`, then you will not be execute from your own directory on Android `>= 10`. Check #2155

Also read https://github.com/termux/termux-app#forking

>how does chroot work?

How about search on the internet or read its docs?
